src/main/java/uk/gov/companieshouse/confirmationstatementapi/contoller/EligibilityController.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.companieshouse.confirmationstatementapi.contoller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestAttribute;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.companieshouse.api.model.company.CompanyProfileApi;
+import uk.gov.companieshouse.confirmationstatementapi.ConfirmationStatementApiApplication;
+import uk.gov.companieshouse.confirmationstatementapi.exception.ServiceException;
+import uk.gov.companieshouse.confirmationstatementapi.model.response.EligibilityFailureResponse;
+import uk.gov.companieshouse.confirmationstatementapi.service.CompanyProfileService;
+import uk.gov.companieshouse.confirmationstatementapi.service.EligibilityService;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+import java.util.Optional;
+
+@RestController
+public class EligibilityController {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ConfirmationStatementApiApplication.APP_NAME);
+
+    @Autowired
+    private CompanyProfileService companyProfileService;
+
+    @Autowired
+    private EligibilityService eligibilityService;
+
+    @GetMapping("/confirmation-statement/company/{company-number}/eligibility")
+    public ResponseEntity<Object> getEligibility(@RequestAttribute("company-number") String companyNumber){
+        LOGGER.debug("Start Handling request  GET '/' for eligibility");
+        CompanyProfileApi companyProfile = null;
+        try {
+            companyProfile = companyProfileService.getCompanyProfile(companyNumber);
+            Optional<EligibilityFailureResponse> validationErrorResponseBody =
```

Comment:
The Spec has been updated since you started on this.
The eligibility endpoint should return a CompanyValidationResponse object regardless of success or failure.
https://github.com/companieshouse/confirmation-statement-api/blob/main/spec/schema.json#L92

Reply:
I cannot find any CompanyValidationResponse in the api. Does this need to be added? If so we should do this on another task.

Reply:
It's not in the api as it needs to be added. Well, really the EligibilityFailureResponse needs to be refactored into CompanyValidationResponse.

Up to you if you do it as part of this pr or as a seperate task, but i'd argue thats a pre-requisite to implementing this endpoint

Reply:
Is this simply a case of renaming? If so I will do it as part of this task.

Reply:
Renaming, and adding `COMPANY_NOT_FOUND` and `COMPANY_VALID_FOR_SERVICE` types to the enum.